lib/chef/provider/package/rubygems.rb

```diff
@@ -604,7 +604,8 @@ def rdoc_string
         end
 
         def needs_nodocument?
-          Gem::Requirement.new(">= 3.0.0.beta1").satisfied_by?(Gem::Version.new(gem_env.rubygems_version))
+          rubygem_version = shell_out!("#{gem_binary_path} --version").stdout.chomp
```

Comment:
I don't want to put another rubygem version method out there just for this case. I think that will cause confusion because we already have 2 similar methods in the same file.